<!DOCTYPE html>
<html lang="en">
<head>
<!-- basic -->
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<!-- mobile metas -->
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="viewport" content="initial-scale=1, maximum-scale=1">
<!-- site metas -->
<title>Analyzing Microbiomes</title>
<meta name="keywords" content="">
<meta name="description" content="">
<meta name="author" content="">
<!-- bootstrap css -->
<link rel="stylesheet" href="css/bootstrap.min.css">
<!-- style css -->
<link rel="stylesheet" href="css/style.css">
<!-- Responsive-->
<link rel="stylesheet" href="css/responsive.css">
<!-- fevicon -->
<link rel="icon" href="images/fevicon.png" type="image/gif" />
<!-- Scrollbar Custom CSS -->
<link rel="stylesheet" href="css/jquery.mCustomScrollbar.min.css">
<link rel="icon" href="images/icon.jpg">

<!-- Tweaks for older IEs-->
<!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script><![endif]-->
</head>
<!-- body -->
<body class="main-layout">
<!-- loader  -->
<div class="loader_bg">
  <div class="loader"><img src="images/loading.gif" alt="#" /></div>
</div>
<!-- end loader --> 
<!-- header -->
<header> 
  <!-- header inner -->
  <div class="head-top">
    <div class="container">
      <div class="row">
        <div class="col-xl-4 col-lg-4 col-md-4 col-sm-4">
          <div class="email"> <a href="#">Email : fascettij21@bcdschool.org</a> </div>
        </div>
        <div class="col-xl-4 col-lg-4 col-md-4 col-sm-4">
          <!--
          <div class="icon"> <i> <a href="#"><img src="icon/facebook.png"></a></i> <i> <a href="#"><img src="icon/Twitter.png"></a></i> <i> <a href="#"><img src="icon/linkedin.png"></a></i> <i> <a href="#"><img src="icon/google+.png"></a></i> </div>
          -->
        </div>
        <div class="col-xl-4 col-lg-4 col-md-4 col-sm-4">
          <!--
          <div class="contact"> <a href="#">Contact :  +71  78908540</a> </div>
          -->
        </div>
      </div>
    </div>
  </div>
  <div class="container">
    <div class="row">
      <div class="col-xl-3 col-lg-3 col-md-3 col-sm-3 col logo_section">
        <div class="full">
          <div class="center-desk">
            <!--
            <div class="logo"> <a href="index.html"><img src="images/logo.jpg" alt="#"></a> </div>
            -->
          </div>
        </div>
      </div>
      <div class="col-xl-9 col-lg-9 col-md-9 col-sm-9">
        <div class="menu-area">
          <div class="limit-box">
            <nav class="main-menu">
              <ul class="menu-area-main">
                <li> <a href="index.html">Home</a> </li>
                <li> <a href="what-are-microbiomes.html">What are Microbiomes?</a> </li>
                <li class="active"> <a href="analyze.html">Analyze</a> </li>
                <li> <a href="impact.html">Impact</a> </li>
              </ul>
            </nav>
          </div>
        </div>
      </div>
    </div>
  </div>
  <!-- end header inner --> 
</header>
<!-- end header -->

<div class="Team-bg">
  <div class="container">
    <div class="row">
      <div class="col-xl-12 col-lg-12 col-md-12 col-sm-12">
        <div class="abouttitle">
          <h2>Analyzing a Microbiome</h2>
        </div>
      </div>
    </div>
  </div>
</div>

<!-- state abouts -->
<div class="abouts"> <dir class="abouts-bg">
  <div class="container">
    <div class="row">
      
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <h3>How do you <strong class="cur">analyze</strong> a microbiome?</h3>
          <p>
            Microbiomes are composed of many different species of microorganisms, and each microbiome (both microbiomes intra- and inter-person) can be vastly different from one another. Thus, being able to determine what an individual’s microbiome is made up of is very important. To do this, the genetic code of the microbes can be sequenced. From a sample, scientists can <a target="_blank" style="font-weight:450;" href="https://www.labome.com/method/DNA-Extraction-and-Purification.html">extract the genetic code</a> and then sequence it to know what the <a style="font-weight:450;" href="what-is-dna.html#dna">DNA</a> is. Typically, researchers that want some microbiome samples sequenced will pay a third party company, send the samples to be sequenced, and receive electronic files with the DNA code from the sample on it. There are two main ways to sequence a microbiomic sample: 16S rRNA and Shotgun Metagenomics. Most of the time, researchers must choose one method over the other for budget reasons, but there are pros and cons to each one.
            
          </p>
        </div>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <br><br><br><br><br><br><br>
          <figure><img style="margin-left:60px" src="images/lab_img2.jpg" alt="img"/></figure>
        </div>
      </div>
      
    </div>
  </div>
  </dir> 
</div>

<div class="abouts"> <dir class="abouts-bg">
  <div class="container">
    <div class="row">
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <br><br>
          <figure><img src="images/###.jpg" alt="img"/></figure>
        </div>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <h3 style="text-transform:inherit;"><strong class="cur">16S rRNA</strong> SEQUENCING</h3>
          <p>
            The 16S <a style="font-weight:450;"href="what-is-dna.html#rna">rRNA</a> gene is the most popular gene sequenced to identify bacteria and other microorganisms. There are other genes that are in bacteria, but the 16S rRNA gene is the most conserved throughout species, meaning it is one of the most reliable to identify the parent organism. The 16s rRNA gene is made up of nine hypervariable regions which are unique to particular species. When performing 16s rRNA sequencing, typically two hypervariable are chosen to identify the spices. The hypervariable regions chosen are given <a target="_blank" style="font-weight:450;" href="https://www.illumina.com/content/dam/illumina-marketing/documents/products/whitepapers/archerdx-molecular-barcode-whitepaper.pdf">molecular barcodes</a> so later computational methods can detect possible errors and identify sequences easily. The sequencing method uses <a target="_blank" style="font-weight:450;" href="https://www.genome.gov/about-genomics/fact-sheets/Polymerase-Chain-Reaction-Fact-Sheet">polymerase chain reaction</a> (PCR) [a] to amplify the selected range of DNA in the sample to a detectable level. Typically, <a target="_blank" style="font-weight:450;" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3841808/">next generation sequencing</a> (NGS), a modern and rapid method for DNA/RNA analysis, is used to sequence the samples. Computational methods are then used against a reference database of known species’ genomes to determine the contents of the sample.  
          </p>
        </div>
      </div>
      
      
    </div>
  </div>
  </dir> 
</div>

<div class="abouts"> <dir class="abouts-bg">
  <div class="container">
    <div class="row">
      
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <h3><strong class="cur">Shotgun Metagenomic</strong> Sequencing</h3>
          <p>
            Shotgun metagenomic sequencing sequences all of the DNA in the sample, then attempts to assign specific parts of the sequence to particular species from a known database. Compared to trying to assign a region a species from a relatively small portion of the genome in 16S, metagenomic sequencing considers the whole genome. This comes with pros and cons. On the up side, it tends to be more reliable and accurate. But on the other hand, it is significantly more expensive. Depending on the study done, researchers may opt for one over the other because of price. Though, the cost of shotgun metagenomic sequencing has been decreasing over time, which may cause an increase in popularity. 
          </p>
        </div>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <br><br>
          <figure><img src="images/###.jpg" alt="img"/></figure>
        </div>
      </div>
      
    </div>
  </div>
  </dir> 
</div>

<div class="abouts"> <dir class="abouts-bg">
  <div class="container">
    <div class="row">
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <br><br>
          <figure><img src="images/###.jpg" alt="img"/></figure>
        </div>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <h3><strong class="cur">Metatranscriptome</strong> Sequencing</h3>
          <p>
            This type of sequencing is foundationally different from both 16S and shotgun metagenomics. The two already mentioned measure the composition of the microbiome in a taxonomic- and census-style. Metatranscriptome sequencing, on the other hand, aims to analyze the microbiome sample for the organisms that are active. For background, microorganisms in the gut microbiome have a connection and interact with the host organism with <a target="_blank" style="font-weight:450;" href="https://doi.org/10.3410/B3-4">chemical signals</a>. Another way that the microorganisms are active is by producing proteins. To do this, they need to go through a process of protein synthesis – one step in which is to produce RNA. Metatranscriptomic sequencing, rather than measuring the DNA, measures the RNA to identify the active microorganisms. This method of sequencing is relatively new, but opens a lot of doors to learning more about microbiomes. A company called <a  target="_blank" style="font-weight:450;" href="https://www.viome.com/">Viome</a> specializes in this type of sequencing. Learn more about one of their founders Dr. Momo Vuyisich’s opinions on the benefits of metatranscriptomic sequencing <a style="font-weight:450;" href="momo.html">here</a>. 
          </p>
        </div>
      </div>
      
      
    </div>
  </div>
  </dir> 
</div>

<div class="abouts"> <dir class="abouts-bg">
  <div class="container">
    <div class="row">
      
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <h3><strong class="cur">Computational</strong> Methods</h3>
          <p>
            Computational power is crucial to the advancement in the field of microbiomes; samples are useless if researches have no way to tell what is in them. One of the most limiting factors in modern computational methods of microbiomic (and general genomic) analysis is the inability to process the sheer amount of data that there could be. After sequencing, the <a target="_blank" style="font-weight:450;" href="ecoli-genome.txt"> electronic file for the genome of escherichia coli</a> [1] [b], more commonly known as e. coli, is over 66,000 lines with 70 nucleotides in each line, and an average microbiome has many trillions of microbes in it. [2] Though a single sample will not have trillions of microorganisms in it, it gives an idea for the scale. There must be significant computational power to sift through all of the data and analyze it. This is part of the reason 16S rRNA sequencing is cheaper than shotgun metagenomic sequencing, since the former only does analysis on a particular gene for each organism and the latter sequences the entire genome.
            <br><br>
            There are bioinformatic [c] pipelines [d] that are free to download like <a target="_blank" style="font-weight:450;" href="http://qiime.org/">QIIME</a> (pronounced <i>chime</i>) and <a target="_blank" style="font-weight:450;" href="https://mothur.org/">Mothur</a>. Each source has their own unique procedure. Once the sequences are acquired, the computational workflow for analyzing a microbiome is different from 16S, to shotgun metagenomics, to metatranscriptomics. For 16S, the hypervariable regions are cross referenced to a database of known species. The computer program can not only look for identical matches, because there will likely be some changes from the sample in question to the reference one. This is most probable to happen either because the gene of the sequenced organism itself had a naturally occurring mutation [e] or there was an error during the sequencing process – it is widely accepted that sequencing is >97% accurate. [3] Since a difference between the reference and unknown sequence is almost guaranteed, it is important that the computer program compensate for that. On the other hand, if the program is too lax in how many differences it allows, it will falsely classify certain 16S genes as a species. There are algorithms that attempt to diminish the computational power needed to do this analysis called dynamic programming. [f] While these methods have made these processes significantly less time and power consuming than brute-force methods, large datasets continue to pose as a challenge for microbiome analysis.  
          </p>
        </div>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <div class="abouts-us">
          <br><br>
          <figure><img src="images/###.jpg" alt="img"/></figure>
        </div>
      </div>
      
    </div>
  </div>
  </dir> 
</div>


<div class="Request">
  <div class="container">
    <div class="row">
      <div class="col-md-12">
        <div class="titlepage3">
          <h2>Footnotes</h2>
        </div>
      </div>
    </div>
    <div class="row">
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <b>[a]</b> Polymerase chain reaction is a method of replicating spesific portions of DNA. Samples undergo a series of heat cycles which activate certain protiens to replicate the selected DNA designated by primers. Learn more about PCR <a target="_blank" style="font-weight:450;" href="https://www.sciencelearn.org.nz/resources/2347-what-is-pcr">here</a>. PCR has gained more public attraction because it is one of the most reliable methods to use for a diagnosis of the novel COVID-19. Learn more about PCR COVID-19 tests <a target="_blank" style="font-weight:450;" href="https://www.fda.gov/consumers/consumer-updates/coronavirus-disease-2019-testing-basics">here</a>.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <b>[b]</b> ecoli
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <b>[c]</b> Bioinformatics is a field of science combining computer science and biology.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <b>[d]</b> A bioinformatic pipline is a computational process used to analyze data. (Ex. microbiomic bioinformatic pipeline)
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <b>[e]</b> Mutation: A change in genetic code, most frequently during DNA replication.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <b>[f]</b> Dynamic programming is a recursive algorithom frequently used in bioinformatic studies. Typically with the input of a .fasta file, it outputs a comparison of two or more sequences in similarities. Learn more <a target="_blank" style="font-weight:450;" href="https://jeffe.cs.illinois.edu/teaching/algorithms/book/03-dynprog.pdf">here</a>.
      </div>

      
      

  </div>
</div>
</div>

<div class="Request">
  <div class="container">
    <div class="row">
      <div class="col-md-12">
        <div class="titlepage3">
          <h2>Sources and links</h2>
        </div>
      </div>
    </div>
    <div class="row">
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
        <a href="https://www.yaleclimateconnections.org/2019/09/how-research-labs-can-become-more-sustainable/" target="_blank">
          [Image 1] https://www.yaleclimateconnections.org/2019/09/how-research-labs-can-become-more-sustainable/
        </a>
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
          <b>[1]</b> <i>Escherichia coli str. K-12 substr. MG1655, complete genome.</i> NCBI. <a target="_blank" style="font-weight:450;" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_000913.3?report=fasta&to=4641652">https://www.ncbi.nlm.nih.gov/nuccore/NC_000913.3?report=fasta&to=4641652</a>.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
          <b>[2]</b> Ursell, L. K., Metcalf, J. L., Parfrey, L. W., & Knight, R. (2012). Defining the human microbiome. <i>Nutrition reviews</i>, 70 Suppl 1(Suppl 1), S38–S44. <a target="_blank" style="font-weight:450;" href="https://doi.org/10.1111/j.1753-4887.2012.00493.x">https://doi.org/10.1111/j.1753-4887.2012.00493.x</a>.
      </div>
      <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
          <b>[3]</b> Goodrich K., J et al. (2014). Leading Edge Primer. <i>Conducting a Microbiome Study</i>. <a target="_blank" style="font-weight:450;" href="https://dx.doi.org/10.1016%2Fj.cell.2014.06.037">https://dx.doi.org/10.1016%2Fj.cell.2014.06.037</a>.
      </div>
      
  </div>
</div>
</div>
  
<!-- end abouts -->
<footer>
  <br>
  <div class="footer">
    <div class="container">
      <div class="row">
        <div class="col-xl-4 col-lg-4 col-md-4 col-sm-12">
          <div class="Contact">
            <h3>
              <a href="sources.html" style="color:white;">
                Resources
              </a>
            </h3>
            <!--
            <ul class="contant_icon">
              <li> <a href="mailto:fascettij21@bcdschool.org"><img src="icon/email.png"></a></li>
            </ul>
            -->
          </div>
        </div>
        <div class="col-xl-4 col-lg-4 col-md-4 col-sm-12">
          <div class="Contact">
            <h3>
              <a href="glossary.html" style="color:white;">
                Glossary
              </a>
            </h3>
            <!--
            <ul class="contant_icon">
              <li> <a href="mailto:fascettij21@bcdschool.org"><img src="icon/email.png"></a></li>
            </ul>
            -->
          </div>
        <!--
        <div class="col-xl-3 col-lg-3 col-md-3 col-sm-12">
          <div class="Social">
            <h3>Social links</h3>
            <ul class="socil_link">
              <li><a href="#"><img src="icon/fb.png"></a></li>
              <li><a href="#"><img src="icon/Tw.png"></a></li>
              <li> <a href="#"><img src="icon/lin.png"></a></li>
              <li> <a href="#"><img src="icon/insta.png"></a></li>
            </ul>
          </div>
        </div>
        <div class="col-xl-5 col-lg-5 col-md-5 col-sm-12">
          <div class="newsletter">
            <h3>newsletter</h3>
            <input class="new" placeholder="Enter your email" type="Enter your email" >
            <button class="subscribe">subscribe</button>
          </div>
        </div>
        -->
      </div>
    </div>
  </div>
  <!--
  <div class="copyright">
    <div class="container">
      <p>Copyright 2019 All Right Reserved By <a href="http://html.design">Free html Templates</a></p>
    </div>
  </div>
  -->
</footer>
<!-- end footer --> 

<!-- Javascript files--> 
<script src="js/jquery.min.js"></script> 
<script src="js/popper.min.js"></script> 
<script src="js/bootstrap.bundle.min.js"></script> 
<script src="js/jquery-3.0.0.min.js"></script> 
<script src="js/plugin.js"></script> 

<!-- sidebar --> 
<script src="js/jquery.mCustomScrollbar.concat.min.js"></script> 
<script src="js/custom.js"></script>
</body>
</html>